### PR TITLE
Changed polyline to work with a list of edges instead of a wire.

### PR DESCRIPTION
**Pull request by jmwright**
This is a rework of the polyline function to fix issue #102 

If someone like @dcowden @hyOzd or @galou wants to look over my code, that would be great. Otherwise I'll just merge after waiting awhile for comments.

### CODE REVIEW

File: cadquery/CQ.py [hunk 1]
```diff
@@ -996,7 +996,7 @@ def _findFromPoint(self, useLocalCoords=False):
```
Comment (galou) [inline cadquery/CQ.py]: WARNING: only the last object in the stack is used.

File: cadquery/CQ.py [hunk 2]
```diff
@@ -996,7 +996,7 @@ def _findFromPoint(self, useLocalCoords=False):
 
         NOTE:
         """
-        obj = self.objects[0]
+        obj = self.objects[-1]
         p = None
```
Comment (galou) [inline cadquery/CQ.py]: The `line p = None` is useless.

File: cadquery/CQ.py [hunk 3]
```diff
@@ -1657,23 +1657,33 @@ def polyline(self, listOfXYTuple, forConstruction=False):
...
-            it is possible to combine a polyline with other edges and arcs
         """
-        vecs = [self.plane.toWorldCoords(p) for p in listOfXYTuple]
-        w = Wire.makePolygon(vecs)
-        if not forConstruction:
-            self._addPendingWire(w)
 
-        return self.newObject([w])
+        # Our list of new edges that will go into a new CQ object
+        p = []
```
Comment (galou) [inline cadquery/CQ.py]: Rename `p` to `edges` to be more expressive.

**Comment by galou** [comment]
I make some comments because I don't know how to change the file directly.

**Comment by jmwright** [comment]
@galou I think you could fork my branch to make changes and then do a PR for that branch, but I'm not exactly sure how/if that works. The comments worked just fine though. Thank you for the feedback. I've incorporated your changes.

**Comment by galou** [comment]
@jmwright I also though I could fork the branch but I don't know how to avoid submitting a new PR when I want the changes to be integrated. I would rather reuse the same PR.

**Pull request merged**